matador/plotting/magres_plotting.py

```diff
@@ -138,7 +139,9 @@ def plot_magres(
         if line_kwargs is not None:
             _line_kwargs.update(line_kwargs[ind])
 
-        relevant_sites = [site for site in doc if site.species == species]
+        _doc = _magres[ind]
+
+        relevant_sites = [site for site in _doc if site.species == species]
```

Comment:
Could you revert these lines and instead just add the underscore to `magres`->`_magres` in the loop on line 128 please? There are quite a few other places that need to be `_doc` instead of `doc` otherwise.